xml/obs_glossary.xml

```diff
@@ -72,30 +72,32 @@
  </glossentry>
  <!-- ############## B ############## -->
  <glossentry xml:id="obs.glos.project_binary">
-  <glossterm>Binary Package</glossterm>
+  <glossterm>Binary Package (Binary)</glossterm>
   <glossdef>
    <para>
-    A binary package (in the &obs; context often called a binary) is
-    an archive file that contains an installable version of software and
+    An archive file that contains an installable version of software and
     metadata. Binary packages can also include, for example, configuration
     files and documentation.
    </para>
    <para>
-    Binary packages are one possible result of building on &obsa;.
-    &obsa; supports building multiple types of binary package. Examples
-    include &rpmf; packages and &debf; packages.
+    Unlike containers, binary packages do not include dependencies of the
+    main software. Instead, dependencies are packaged as additional binary
+    packages. Formats of binary packages include &rpmf; and &debf;.
    </para>
    <para>
-    Alternatively, &obsa; can also build operating system images and
-    containers.
+    You can build binary packages on &obsa;.
    </para>
    <remark>
     "Binary package" poses a bit of a question: It is a bit unclear when a
-    package can be considered binary: 1) all archive formats are pretty
-    much always binary-based; 2) interpreted languages actually aren't
-    binary. The term "installable package" comes to mind, however, but it
-    is definitely not in common usage yet, unlike binary package.
-    - sknorr, 2017-08-23
+    package can be considered binary: 1) all archive formats are pretty much
+    always binary-based; 2) interpreted languages actually aren't binary. The
+    term "installable package" comes to mind, however, but it is definitely
+    not in common usage yet, unlike binary package.
+    Additionally, the usage of "binary" to mean "binary package" in OBS docs
+    is problematic. Everywhere else, a binary is defined as a single
+    executable file (though whether that file is binary or actually
+    human-readable like a script, usually does not matter). - sknorr,
+    2017-08-30
```

Comment:
I would propose to complete avoid the term "binary package" altogether. This just leads to problems. Maybe we could mention this inside a parenthesis as an additional term, but we should "ban" this term for the reason you've stated above.